src/Microsoft.AspNet.Server.Testing/Deployers/IISDeployer.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if DNX451
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using Microsoft.Framework.Logging;
+using Microsoft.Web.Administration;
+
+namespace Microsoft.AspNet.Server.Testing
+{
+    /// <summary>
+    /// Deployer for IIS (Both Helios and NativeModule).
+    /// </summary>
+    public class IISDeployer : ApplicationDeployer
```

Comment:
File a work item to see if we can move the IIS specific deployers to the Helios repo. It will be easier to maintain them that way.